### DISCUSSION TITLE
Wayland quá dở

### DISCUSSION THREAD

**Discussion by HieuNGN** [BambooEngine/ibus-bamboo#511] [General]
Tui viết cái này để tương lai ông nào có dùng Fedora mà chạy Wayland làm giao diện thì chuyển nhanh sang Xorg (X11). 

Đau đầu mất mấy ngày vì mò mãi không ra lí do không gõ được telex trên Thorium hoặc các browsẻr Chromium. Đến hôm nay chợt nhớ ra là mọi người ai cũng bảo Wayland như c*t nên thử chạy bằng X11. Bùm, problem solved. 

Anh em author nên note cái này lại trong readme để tránh trường hợp đần độn của tôi ạ huhu  T_T

**Comment by goatastronaut0212** [comment]
@HieuNGN 

Mình nghĩ là các upstream developers như Wayland hay Ibus cũng không được trả đủ tiền cho việc hỗ trợ input method protocol trên Wayland lắm. Họ là những developers làm vì sở thích nên bạn đừng quá gắt gao trong chuyện đó nên lần sau bạn hãy chú ý việc sử dụng từ nếu bạn có ý kiến nào đó.

Wayland còn chưa ổn định lắm nên tạm thời mọi người hãy tiếp tục sửa dụng Xorg.

Cảm ơn vì bạn đã nhắc mình sẽ note lại.

**Comment by hien-ngo29** [comment]
@phamthaison1207 Mình dám khẳng định Linux có vô vàn thứ phức tạp hơn Windows rất nhiều lần. Tại sao bộ gõ trên Linux lại không ổn định bằng Unikey/EVKey, tất cả đều nằm ở triết lý bảo mật của linux nói chung và wayland nói riêng, gây nhiều khó khăn trong việc thiết kế bộ gõ. Thế giới của Linux rất rộng lớn, có lẽ bạn chỉ thấy được những cái hại của nó và chưa thấy cái lợi. Thứ mình muốn thấy là một câu hỏi giúp đỡ hơn là một phát biểu mang tính tiêu cực (bạn nói "dành cho Tây lông thích hype" thực sự rất khó chịu), bạn làm như vậy chẳng khác gì muốn rage bait cộng đồng người dùng linux, dẫn tới cuộc trò chuyện trở nên toxic là đúng rồi.

Như mọi người nói bộ gõ như fcitx-unikey (và ibus-lotus là fork của ibus-bamboo do mình làm, tuy còn vài vấn đề nhưng mình đã vá được kha khá vấn đề trên Wayland) thực chất đã hỗ trợ khá tốt và không có quá nhiều vấn đề. Mình cũng xin lỗi nếu có hơi quá lời một chút vì có thể bạn chỉ nói lên những khó chịu về việc dùng Linux thôi, Linux thực chất có rất nhiều điểm cộng mà bạn chưa biết, đặc biệt là về hiệu suất và tính tùy biến. Có lẽ bạn không biết nhưng các sản phẩm của công ty game huyền thoại Valve như máy chơi game Steam Deck, Steam Machine,... đều dùng Linux (SteamOS) vì lý do như vậy. Nếu bạn có ý định dùng Linux nhưng ngại tìm hiểu thì đừng nên dùng, hãy luôn tìm tòi khi bạn là một người dùng Linux bạn nhé.

**Reply by nhktmdzhg** [reply]
ông bạn có nhầm lẫn giữa review và quảng cáo không? Các distro Linux cho desktop 99.99% là phi lợi nhuận và hoạt động vì cộng đồng, quảng cáo để được cc gì, ai trả tiền cho mà quảng cáo? Còn linux có nhiều thứ ngon hơn windows đó rõ ràng là fact, người ta có quyền nói chứ, sao lại cấm, giờ tôi bảo dùng linux vì quyền riêng tư thì ông phản bác hộ tôi xem, tôi dùng linux để ricing thì sao, dùng linux vì ăn ram ít thì sao, dùng vì ổn định thì sao, review thằng này chê thằng khác là cái chuyện bình thường trên đời, quái gì trong mắt bạn nó thành tệ hại? Không xài được thì qua win, qua mac mà xài, phải vào hẳn 1 repo đã ngừng hoạt động chỉ để toxic thì anh bạn thực sự ngầu đó? Còn nếu bạn thấy nó không dành cho người Việt xài thì như trên, xóa đi, để những người Việt khác người ta có nhu cầu người ta xài, chả đến lượt ông phát biểu về "linux cho tây lông", toxic chúa, lôi cả bên tây vào trong khi đ liên quan mịa j, phương đông như trung quốc nó cũng xài linux và đóng góp ào ào kìa, chính cái ibus ông đang chửi cũng là của trung quốc đấy.

**Comment by ShotChannel** [comment]
Wayland thì dùng Fcitx đi bác.

**Comment by goatastronaut0212** [comment]
Mình thực sự để account này inactive 1 thời gian vì 1 vài thông báo mình mới quay lại. Mình sẽ khóa discussion lại vì nó hơi thành 1 cuộc khẩu chiến hơi vô nghĩa. Thực tế nếu ibus-bamboo có tệ mình đã thừa nhận chỉ đơn giản là nó xây trên 1 nền móng không vững vàng nhưng bảo cộng đồng Linux không có giải pháp như fcitx5 hay fcitx5-unikey thì là hơi xuyên tạc thiếu khách quan. Hy vọng bất cứ ai xem lại discussion này hãy đọc file README của dự án trước khi gõ bàn phím thêm. Cảm ơn.
